packages/backend/src/app/database/seeds/dev-seeds.ts

```diff
@@ -0,0 +1,39 @@
+import { ApEnvironment } from "@activepieces/shared";
+import { authenticationService } from "../../authentication/authentication.service";
+import { logger } from "../../helper/logger";
+import { system } from "../../helper/system/system";
+import { SystemProp } from "../../helper/system/system-prop";
+import { userService } from "../../user/user-service";
+
+const seedDevUser = async (): Promise<void> => {
+    const devEmail = "dev@ap.com";
+    const devPassword = "12345678";
+    const devUser = await userService.getOneByEmail({ email: devEmail });
+
+    if (!devUser) {
+        await authenticationService.signUp({
+            email: devEmail,
+            password: devPassword,
+            firstName: "firstName",
+            lastName: "lastName",
+            trackEvents: false,
+            newsLetter: false,
+        });
+    }
+
+    logger.info(`[seedDevUser] email=${devEmail} pass=${devPassword}`);
```

Comment:
make it human readable, like You can use the following account to sign in

Reply:
added a warning message in cbf594b